storage/postgres/queries.go

```diff
@@ -0,0 +1,118 @@
+package postgres
+
+const namespaceListQuery = `
+SELECT id from namespaces
+`
+
+const namespaceGetQuery = `
+SELECT * from namespaces where id=$1
+`
+
+const namespaceDeleteQuery = `
+DELETE from namespaces where id=$1
+`
+
+const namespaceUpdateQuery = `
```

Comment:
We are not accepting compatibility anymore during namespace creation. also want to discuss why format should be both places in namespace and schema

```
message CreateNamespaceRequest {
  string id = 1 [(google.api.field_behavior) = REQUIRED];
  Schema.Format format = 2 [(google.api.field_behavior) = REQUIRED];
  string description = 3;
}
```

Reply:
For all namespace queries.

Reply:
I have added compatibility in db schema because we are going to have it. Plan is to get the APIs ready then implement compatibility logic. Thought of adding compatibility field that time.